.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+    branches: [ master, main ]
+  pull_request:
+    branches: [ master, main ]
+    
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: makeappdev/uselatex:latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Fetches all history for all branches and tags
+
+      - name: Configure Git safe directory with GITHUB_WORKSPACE
+        run: git config --global --add safe.directory $GITHUB_WORKSPACE
```

Comment:
Let's wait for the outcome of [this discussion](https://github.com/latex4ei/CheatsheetTemplate/pull/2/files#r1542612844).